src/pmi/pmi.cs

```diff
@@ -122,7 +122,7 @@ public static Assembly Resolve(string assemblyName, AssemblyLoadContext context)
     }
 }
 
-#if NETCOREAPP3_0 || NETCOREAPP3_1
+#if NETCOREAPP3_0 || NETCOREAPP3_1 || NETCOREAPP5_0
```

Comment:
Couldn't we reverse this, so the netcoreapp2.2 (?) and below (in the `else` clause) get the `#if` and 3.0+ get the `#else` and thus don't need to get updated going forward?

Reply:
Sure.